packages/dashboard/src/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -223,8 +291,10 @@ function GoogleAnalyticsSettings({
 }
 
 GoogleAnalyticsSettings.propTypes = {
-  handleUpdate: PropTypes.func,
   googleAnalyticsId: PropTypes.string,
+  handleUpdateAnalyticsId: PropTypes.func,
+  usingLegacyAnalytics: PropTypes.bool,
```

Comment:
Should we set default for `usingLegacyAnalytics` to false?

Reply:
The value is coming from the settings provider and the default is already `false` in `defaultSettingsState`. So this is already covered.

Reply:
If you had default to false, you wouldn't have had to update lots of the tests.